schema/cp1/metar.json

```diff
@@ -0,0 +1,110 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://github.com/iblsoft/swimdemo/schema/metar.json",
+    "title": "MET-SWIM AMQP CP1 Message - METAR/SPECI",
+    "type": "object",
+    "additionalProperties": false,
+    "properties": {
+        "address": {
+            "type": "string",
+            "enum": ["weather.aviation.metar"]
+        },
+        "priority": { "type": "integer", "minimum": 4, "maximum": 6 },
+        "subject": { "type": "string", "const": "weather.aviation.metar" },
+        "content-type": { "type": "string", "const": "application/xml" },
+        "content-encoding": { "type": "string", "enum": ["gzip", "identity"] },
+        "absolute-expiry-time": { "type": "integer", "minimum": 0 },
+        "creation-time": { "type": "integer", "minimum": 0 },
+        "conformsTo": { "type": "string", "format": "uri" },
+
+        "properties.issue_datetime": {
```

Comment:
I also used a flat structure first but I think it might make sense to build a nested structure here (especially with respect to future headers, e.g., for messages containing a link), i.e., building a properties object and then having fields `issue_datetime`, etc therein.